src/scenes/Oracle/components/PredictionAlert/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { PropTypes } from 'react';
+import Button from 'material-ui/Button';
+import Dialog, {
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from 'material-ui/Dialog';
+
+class PredictionAlert extends React.PureComponent {
```

Comment:
Name it something more specific like `TransactionCompleteDialog` or something of that sort? Since were not using antd anymore its a different component name.